### PR TITLE
Feature portfolio accessibility

### PR DESCRIPTION
**Pull request by MahdiAghaali**
Accessibility report and changes:

- Page titles: changed the title
- Image text alternatives: added alt text
- Text headings: no issues found
- Color contrast: changed the foreground color of orange buttons
- Resize: no issues found
- Interaction: added focusable links to navbar for keyboard accessibility
- Moving content: no issues found
- Multimedia: no multimedia present
- The basic structure of the page: no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,18 +23,19 @@
             <i id="burguer-menu" class="fa-solid fa-bars"></i>
             <div id="desktop-menu" class="div-flex">
               <ul id="desktop-ul-menu" class="div-flex">
-                <li><a>Portfolio</a></li>
-                <li><a>About</a></li>
-                <li><a>Contact</a></li>
+                <li><a href="#wellcome-section">Portfolio</a></li>
+                <li><a href="#about-section">About</a></li>
+                <li><a href="contact-form">Contact</a></li>
```
Comment (MahdiAghaali) [inline index.html]: - Interaction: added focusable links to navbar for keyboard accessibility

File: index.html [hunk 2]
```diff
@@ -23,18 +23,19 @@
...
         </nav>
-        <section class="welcome-section div-flex">
+        <section id="wellcome-section" class="welcome-section div-flex">
           <div class="welcome-text">
-            <h1>Hey There<span>.</span><br> I'm Carrera</h1>
-            <h2>I'm a Full Stack Developer</h2>
-            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            <h1>Hey There<span>.</span><br> I'm Carrera
+            <br><h2>I'm a Full Stack Developer</h2>
+            </h1>
```
Comment (MahdiAghaali) [inline index.html:37]: Text headings: nested h2 inside h1

File: index.html [hunk 3]
```diff
@@ -43,10 +44,10 @@ <h2>I'm a Full Stack Developer</h2>
                 <li><a href="https://twitter.com/carreraprog"><i class="fa-brands fa-twitter"></i></a></li>
             </ul>
             <div class="illustration-header-mobile div-flex">
-                <img class="vectors" src="vectors/illustration-header-mobile.png" alt="illustration-header">
-            </div>      
+                <img class="vectors" src="vectors/illustration-header-mobile.png" alt="illustration header mobile">
+            </div>
```
Comment (MahdiAghaali) [inline index.html:55]: Updated alt text

File: styles.css [hunk 4]
```diff
@@ -639,9 +640,9 @@ footer {
   align-items: center;
   padding: 12px;
   gap: 10px;
-  background: #ff6b00;
+  background: var(--red);
   margin-top: 24px;
-  color: white;
+  color: var(--black);
```
Comment (MahdiAghaali) [inline styles.css:645]: Changed foreground color for orange buttons for better color contrast

**Review by aimemalaika — APPROVED** [review]
Hi @MahdiAghaali , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**